tests/unit/src/flixel/ui/FlxButtonTest.hx

```diff
@@ -14,6 +17,7 @@ class FlxButtonTest extends FlxTest
 	{
 		button = new FlxButton();
 		destroyable = button;
+		FlxG.mouse.playback(new MouseRecord(0, 0, FlxInputState.RELEASED, 0)); // put the mouse in the upper-left corner to prepare for highlight testing
```

Comment:
As a programmer I would, but as a user, having the cursor jump to 0,0 even when I am resetting a game might be unexpected.